package.json

```diff
@@ -13,8 +13,8 @@
     "build": "pnpm turbo run build",
     "dev": "pnpm dev --prefix src/web/app --",
     "develop": "pnpm dev",
-    "eslint": "TIMING=1 eslint --config .eslintrc.js --ignore-path .gitignore \"./src/backend/**/*.js\" \"./test/**/*.js\"",
-    "eslint-fix": "eslint --config .eslintrc.js --ignore-path .gitignore \"./src/backend/**/*.js\" \"./test/**/*.js\" --fix",
+    "eslint": "TIMING=1 eslint --ignore-path .gitignore",
```

Comment:
If we're removing the `eslintrc` from the root, do we need these?  cc @menghif

Reply:
We can remove both `eslint` and `eslint-fix`.